src/main/kotlin/com/zopa/ktor/opentracing/OpenTracingClient.kt

```diff
@@ -70,4 +70,4 @@ class OpenTracingClient {
             }
         }
     }
-}
```

Comment:
How do you feel about adding .editorconfig to prevent this kind of thing happening in PRs?

Reply:
Yes, good idea! Feel free to add one :)

Reply:
Added :)